file-formats/doma/type/zif_aff_doma_v1.abap

```diff
@@ -0,0 +1,112 @@
+INTERFACE zif_aff_doma_v1 PUBLIC.
+
+  "! <p class="shorttext">Plus - Minus sign for domains</p>
+  "! Plus - Minus sign for domains
+  TYPES ty_sign                 TYPE c.
```

Comment:
Should we add an enumeration for plus and minus here?

Reply:
Would be something like the example below be ok?
`  "! <p class="shorttext">Sign</p>
  "! Sign
  "! $values {@link zif_aff_doma_v1.data:co_sign}
  TYPES ty_sign TYPE c LENGTH 1.

  CONSTANTS:
    "! <p class="shorttext">Sign</p>
    "! Sign
    BEGIN OF co_sign,
      "! <p class="shorttext">Plus</p>
      "! Plus
      Plus TYPE ty_sign VALUE '+',
      "! <p class="shorttext">Minus</p>
      "! Minus
      Minus TYPE ty_sign VALUE '-',
    END OF co_sign.`

Reply:
Would something like the example above be ok?

Reply:
Yes, in pricinple, I think this is the way to go. Usually, we take the values which are stored on the database (as far as I can see this would be `x` and `<space>` in this case. Therefore, I think I would end up with a proposal like this:

```
  CONSTANTS:
    "! <p class="shorttext">Sign</p>
    "! Sign
    BEGIN OF co_sign,
      "! <p class="shorttext">Plus</p>
      "! Plus
      Plus TYPE ty_sign VALUE ' ',
      "! <p class="shorttext">Minus</p>
      "! Minus
      minus TYPE ty_sign VALUE 'X',
    END OF co_sign.
```

Another option would be that we used just a boolean to specify whether the field allows a sign (i.e., negative values) or not. In this case we wouldn't need constants. What do you think?

Reply:
While the boolean option is indeed shorter, by quickly looking at the code, I think the constant one is easier to understand.

Reply:
If we move it to a boolean, we most probably will have also to rename the field to something like `negative_values_supported`(or similar). 

However, I'll reach out to some DDIC experts and ask whether they have an opinion on this. It might be that it take some time until I get a response due to the Holiday season.

Reply:
I have got already feedback by the DDIC experts. They would prefer the boolean option so that we can see in the JSON following information:

```
  „supportsNegativeValues“: true
```

What do you think? Is this fine for you?

Reply:
```suggestion
  TYPES ty_negativeValues TYPE abap_bool.
```

Its fine by me, would this be ok?

Reply:
Almost. Just an underscore is missing :)
```suggestion
  TYPES ty_negative_values TYPE abap_bool.
```

Reply:
thanks!